conf/application.conf

```diff
@@ -47,6 +47,11 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
 # Json error handler
 play.http.errorHandler = "controllers.CustomJsonErrorHandler"
 
+play.filters.enabled += "play.filters.cors.CORSFilter"
```

Comment:
This seems to be for local testing?

Reply:
yes. do we need to remove it or can keep it here?

Reply:
I'd remove it